en/docs/how-tos/security/api-authentication/secure-apis-using-oauth2.0-access-tokens/secure-apis-using-jwt-self-contained-jwt.md

```diff
@@ -5,13 +5,14 @@ Microgateway can accept JWTs issued by a **trusted** key manager as a valid toke
 When a JWT is used as an access token, API Microgateway validates the following attributes/claims of the JWT.
 
 -   **Signature** - after validating the signature, API Microgateway checks whether the JWT is issued by a trusted key manager, and JWT has not tampered in the middle. This signature validation is done by using the public certificate of the key manager which issued the JWT. Importing the public certificate into the API Microgateway trust store and configuring the certificate alias in the JWT validation config section is explained in the [importng certificates to the microgateway truststore]({{base_path}}/how-tos/security/importing-certificates-to-the-api-microgateway-truststore/)
--   **Issuer(iss)** - The issuer claim is a mandatory claim when JWT is used as a security token. Microgateway validates the **iss** claim present in the JWT against the issuer provided in the **jwtTokenConfig** section of the configuration
--   **Audience (aud)** - The audience claim is also a mandatory claim for a JWT that is used as a security token. Microgateway validates the **aud** claim present in the JWT against the audience provided in the **jwtTokenConfig** section of the configuration. But if the audience is not provided in the **jwtTokenConfig** then Microgateway will not mandate the validation of audience claim.
+-   **Issuer(iss)** - The issuer claim is a mandatory claim when JWT is used as a security token. Microgateway validates the **iss** claim present in the JWT against the issuer provided in the **jwtTokenConfig** section of the configuration.
+-   **Subject(sub)** - The subject claim is also a mandatory claim in a token. Microgateway uses the value of the **sub** claim as the user of the secured API.
+-   **Audience(aud)** - The audience claim is also a mandatory claim for a JWT that is used as a security token. Microgateway validates the **aud** claim present in the JWT against the audience provided in the **jwtTokenConfig** section of the configuration. But if the audience is not provided in the **jwtTokenConfig** then Microgateway will not mandate the validation of audience claim.
```

Comment:
Lets say this is not mandatory. It can be made mandatory by adding it to the tokenConfig